Source/WebCore/platform/graphics/ImageOrientation.h

```diff
@@ -157,23 +172,17 @@ struct ImageOrientation {
     Orientation m_orientation { None };
 };
 
-} // namespace WebCore
-
-namespace WTF {
-
-template<> struct EnumTraits<WebCore::ImageOrientation::Orientation> {
-    using values = EnumValues<
-    WebCore::ImageOrientation::Orientation,
-    WebCore::ImageOrientation::Orientation::FromImage,
-    WebCore::ImageOrientation::Orientation::OriginTopLeft,
-    WebCore::ImageOrientation::Orientation::OriginTopRight,
-    WebCore::ImageOrientation::Orientation::OriginBottomRight,
-    WebCore::ImageOrientation::Orientation::OriginBottomLeft,
-    WebCore::ImageOrientation::Orientation::OriginLeftTop,
-    WebCore::ImageOrientation::Orientation::OriginRightTop,
-    WebCore::ImageOrientation::Orientation::OriginRightBottom,
-    WebCore::ImageOrientation::Orientation::OriginLeftBottom
-    >;
+enum class SerializableOrientation : int {
```

Comment:
This isn't great.  Why not just make enum Orientation a nested enum class?

Reply:
Proposing https://github.com/WebKit/WebKit/pull/9236 instead - will pull it out of this change.